### PR TITLE
Glance quotas

### PR DESCRIPTION
**Pull request by stelucz**
Add parameters for quota options

Quota options are parametrized. Each quota can be optionaly specified otherwise default values are used. If parameter doesn't have value, default value is used.

Hello guys @fpytloun @epcim could you please review this request. Thank you very much.

Local kitchen test passed.

### CODE REVIEW

**Comment by stelucz** [comment]
Hello guys @fpytloun @epcim could you please review this request. Thank you very much.

File: glance/files/kilo/glance-api.conf.Debian [hunk 1]
```diff
@@ -304,20 +304,25 @@ scrubber_datadir = /var/lib/glance/scrubber
 
 # The maximum number of image members allowed per image
 #image_member_quota = 128
+{% if server.quota is defined %}{% if server.quota.image_member_quota is defined %}image_member_quota = {{ server.quota.image_member_quota|default('128') }}{% endif %}{% endif %}
```
Comment (fpytloun) [inline glance/files/kilo/glance-api.conf.Debian]: Put conditionals on separate lines to be more readable and use `{%-` to get rid of empty lines

File: README.rst [hunk 2]
```diff
@@ -127,6 +127,21 @@ RabbitMQ HA setup
...
 
+Quota Options
+
+.. code-block:: yaml
+
+    glance:
+      server:
+        ....
+        quota:
+          image_member_quota: -1
```
Comment (fpytloun) [inline README.rst]: I would suggest using this form:
```
quota:
  image_member:
  image_property:
```

etc. because quota is redundant then.

**Comment by stelucz** [comment]
Thank you for review.

**Pull request merged**